daemon/api_validate_test.go

```diff
@@ -599,12 +599,27 @@ func (s *apiValidationSetsSuite) TestGetValidationSetPinnedNotFound(c *check.C)
 }
 
 func (s *apiValidationSetsSuite) TestApplyValidationSetMonitorModePinnedLocalOnly(c *check.C) {
```

Comment:
shouldn't these tests check for the result now? also we return the result for Enforce too, so those tests should check too?